doc/api/fs.md

```diff
@@ -846,10 +870,21 @@ If `options` is a string, then it specifies the encoding. Example:
 fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 ```
 
-Any specified file descriptor has to have been opened for appending.
+The `file` may be specified as a numeric file descriptor that has been opened
+for appending (using `fs.open()` or `fs.openSync()`. It is important to note
```

Comment:
needs closing parenthesis after `openSync()`

Reply:
\+ The same for `fs.appendFileSync()` note.